examples/step-62/step-62.cc

```diff
@@ -1371,10 +1374,100 @@ int main(int argc, char *argv[])
 
       Utilities::MPI::MPI_InitFinalize mpi_initialization(argc, argv, 1);
 
-      HDF5::File  data_file("results.h5",
+#ifdef USE_JUPYTER_NOTEBOOK
+      HDF5::File data_file("results.h5",
                            HDF5::File::FileAccessMode::open,
                            MPI_COMM_WORLD);
-      HDF5::Group data = data_file.open_group("data");
+      auto       data = data_file.open_group("data");
+#else
+      // If we don't use the jupyter notebook we have to create the groups and
+      // set the parameters.
+      HDF5::File data_file("results.h5",
+                           HDF5::File::FileAccessMode::create,
+                           MPI_COMM_WORLD);
+      auto       data = data_file.create_group("data");
+
+      std::vector<std::string> group_names = {"displacement", "calibration"};
+      for (auto group_name : group_names)
+        {
+          // Create the group
+          auto group = data.create_group(group_name);
+
+          // Dimensions of the domain
+          // The waveguide length is equal to dimension_x
+          group.set_attribute<double>("dimension_x", 2e-5);
+          // The waveguide width is equal to dimension_y
+          group.set_attribute<double>("dimension_y", 2e-8);
+          // Position of the probe that we use to measure the flux
+          group.set_attribute<double>("probe_pos_x", 8e-6);
```

Comment:
This is more of a style suggestion. It turns out that the tutorials become difficult to read if you alternate between commentary and code too often. It's easier to read if you keep more of the code and more of the documentation together. For example, at the top (line 1393) you could do something like this:
```
// For each of these two group names, we now open the group and put attributes into these groups.
// Specifically, these are:
// - The dimensions of the waveguide (in $x$ and $y$ directions)
// - The position of the probe (in $x$ and $y$ directions)
...
```
Then you can just show the whole code block at once.